packages/wallet/src/data/api/routers/assets.ts

```diff
@@ -865,6 +850,53 @@ function map(data: {
 
   const lows = priceHistory.map(({ low }) => low).filter(low => low > 0)
 
+  // Use price data as fallback when tokenMetadata is missing or has default values
```

Comment:
I think there is a typo here
```suggestion
  // Use price data as fallback when tokenMetadata is missing or as default values
```